libraries/AP_DDS/dds_xrce_profile.xml

```diff
@@ -31,7 +31,7 @@
     </topic>
   </data_writer>
   <topic profile_name="navsatfix0__t">
-    <name>rt/ROS2_NavSatFix0</name>
```

Comment:
While we change these, let's take out the `0`, since that's now in the frame ID.